src/wallets/IdentityWallet.ts

```diff
@@ -341,6 +341,7 @@ export class IdentityWallet implements TLWallet {
       rawTx.gasPrice,
       rawTx.gasLimit
     )
+    rawTx.feeRecipient = rawTx.feeRecipient || metaTxFees.feeRecipient
```

Comment:
Can it not be, that `metaTxFees.feeRecipient` is also undefined?
Should it not be `rawTx.feeRecipient || metaTxFees.feeRecipient || '0x000...'`

Reply:
I can add it, but it is already part of the relay (since the beginning I guess) that it has to return with feeRecipient: https://github.com/trustlines-protocol/relay/blob/master/src/relay/api/schemas.py#L76

Reply:
Ah okay. Well to be complete I would say it should fill in the 0x0 address as default. At least this is what I would expect if the relay server would not specify a fee recipient.